feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/data/subscription/RealPushSubscriptionService.kt

```diff
@@ -31,6 +34,14 @@ import kotlinx.coroutines.tasks.await
 private const val COLLECTION_NAME = "users"
 private const val PREFS_FIRESTORE_UUID = "firestore_uuid"
 
+private const val GOV_STATE_TOPIC_NAME = "govState"
+private const val NEW_REFERENDA_TOPIC_NAME = "govNewRef"
+
+class TrackIdentifiable(val chainId: ChainId, val track: BigInteger) : Identifiable {
+    override val identifier: String
+        get() = "$chainId:$track"
```

Comment:
Looks like it is better to make it a standard property instead - it will be accessed anyway but we dont wont to recompute it each time